### PR TITLE
Development1

### PR DESCRIPTION
**Pull request by shubambhasin**

### CODE REVIEW

**Comment by Sohail737** [comment]
Great code, just found one small thing,
In VideoCard Component on line 24, addToHistory function is taking state as a parameter but it is not used anywhere, other than that good job

**Comment by shubambhasin** [comment]
will fix it

File: src/components/Player.js [hunk 1]
```diff
@@ -1,87 +1,141 @@
...
+    // } else {
+    //   dispatch({ type: REMOVE_FROM_SAVED_VIDEOS, payload: data });
+    // }
+    try {
+      const { res } = axios.post(
+        "https://videolibrarybackend.shubambhasin.repl.co/saved",
+        data
+      );
+      const resp = res;
+      console.log(resp);
```
Comment (Ashish-j-shetty) [inline src/components/Player.js]: Remove this console.log

File: src/reducer/actions.js [hunk 2]
```diff
@@ -9,33 +10,44 @@ export const REMOVE_FROM_UNLIKED_VIDEOS = "REMOVE_FROM_UNLIKED_VIDEOS";
 export const ADD_TO_PLAYLIST = "ADD_TO_PLAYLISTS";
 
 export const isVideoInSaved = (state, video) => {
-  if (state.savedVideos.filter((data) => data.id === video.id).length === 0) {
+  if (state.savedVideos.filter((data) => data.videoId === video.videoId).length === 0) {
     return false;
   } else {
```
Comment (Ashish-j-shetty) [inline src/reducer/actions.js:15]: You don't actually need an else block here as you are already returning from if block you can directly return true.

Comment (shubambhasin) [reply src/reducer/actions.js:15]: Will do the changes Thanks !

**Pull request merged**